confd/templates/atdd-staging/kilda.properties.tmpl

```diff
@@ -1,3 +1,6 @@
+# Generated by confd.
+# Do not change this file, all changes will be lost. Change corresponding template.
```

Comment:
This is already a template, isn't it?

Reply:
Yes... my goal to have this notice into generated file, so it will be obvious that it was generated from template.

I am not sure that I understood your question.

Reply:
Mmm, ok, I got your point. But at the same time when you open the template file you see "Generated by confd..do not change this file.." header which is very misleading